sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/AnalyticsTransmissionTarget.java

```diff
@@ -46,6 +55,23 @@ public void trackEvent(String name, Map<String, String> properties) {
         Analytics.trackEvent(name, properties, this);
     }
 
+    /**
+     * Create a new transmission target based on the properties of the current target.
+     *
+     * @param transmissionTargetToken The transmission target token of the new transmission target.
+     * @return The new transmission target.
+     */
+    public synchronized AnalyticsTransmissionTarget getTransmissionTarget(String transmissionTargetToken) {
+
+        /* Reuse instance if a child with the same token has already been created. */
+        AnalyticsTransmissionTarget childTarget = mChildrenTargets.get(transmissionTargetToken);
```

Comment:
There is one edge case covered in Apple, basically if transmissionTargetToken and mTransmissionTargetToken are the same then getTransmissionTarget returns self. It's opened to discussion.

Reply:
So we won't do anything actually in that case so this implementation is good. The nested target having the same token can still have a different behavior than its parent.